include/Peer.hpp

```diff
@@ -42,6 +44,7 @@ class Peer
 		std::string		_user;
 		std::string		_realname;
 		std::string		_mode;
+		std::string		_user_pwd;
```

Comment:
`pwd` is the worst name you could think of for password (too similar to **p**arent **w**orking **d**irectory)

Reply:
`vfurmane` is the worst name you could think of

Reply:
Imagine thinking that I chose my name. This is so offensive and white men should be punished for their oppression over the centuries... :roll_eyes:

Reply:
"Pierre Piques".

Are you 143 years old ?